setuptools/command/bdist_wheel.py

```diff
@@ -595,3 +593,17 @@ def adios(p: str) -> None:
             shutil.copy(license_path, os.path.join(distinfo_path, filename))
 
         adios(egginfo_path)
+
+    def _zip_compression(self) -> int:
+        if (
+            isinstance(self.compression, int)
+            and self.compression in self.supported_compressions.values()
+        ):
+            return self.compression
+
+        if isinstance(self.compression, str) and (
+            compression := self.supported_compressions.get(self.compression)
+        ):
+            return compression
```

Comment:
I am really puzzled by the error in the CI:

```
>       raise ValueError(f"Unsupported compression: {self.compression!r}")
E       ValueError: Unsupported compression: 'stored'
```

Maybe ZIP_STORED is 0?

Let's see if the following change fixes it:

```suggestion
        compression = self.supported_compressions.get(str(self.compression))
        if compression is not None:
            return compression
```